docs/content/get-started/upload-launch.md

```diff
@@ -7,13 +7,22 @@ This has 3 steps:
 * Launch a cloud deployment
 
 
-### Set your project name
+### Set your SpatialOS project name
 
-When you signed up for SpatialOS, your account was automatically associated with an organisation and a project, both of which have the same generated name. To find this name enter the [Console](https://console.improbable.io/projects). It should looks like `beta_randomword_anotherword_randomnumber`:
+When you signed up for SpatialOS, your account was automatically associated with an organisation and a project, both of which have the same generated name. 
 
-<img src="{{assetRoot}}assets/project-page.png" style="margin: 0 auto; display: block;" />
-
-Using a text editor of your choice, open `gdk-for-unity-fps-starter-project/spatialos.json` and replace the `unity_gdk` project name with the project name you were assigned in the Console. This will tell SpatialOS which project you intend to upload to. Your `spatialos.json` should look like this:
+1. Find this name by going to the [Console](https://console.improbable.io/projects). <br/>
+The name should look something like **beta_randomword_anotherword_randomnumber**.
+In the example below, it's **beta_yankee_hawaii_621**.
+<br/>
+<br/>
+![]({{assetRoot}}assets/project-page.png)<br/>
+_Example autogenerated SpatialOS project name_
+<br/>
+<br/>
+2. Using a text editor of your choice, open `gdk-for-unity-fps-starter-project/spatialos.json` and replace the `unity_gdk` project name with the project name you were assigned in the Console. This tells SpatialOS which SpatialOS project you intend to upload to. 
+<br/><br/>
+Your `spatialos.json` should look like this:
```

Comment:
that code snippet is actually outdated as well due to the sdk_version that has changed. Not sure whether we want to point that out or if that's obvious

Reply:
Oh, bums. I've made and accepted changes via the suggestion tool. Shall we leave it for this PR and add a change in the next one (with the third bug)?